src/questions/introduction_to_defi.yaml

```diff
@@ -0,0 +1,1147 @@
+- uuid: 4d404192-3ea7-4994-a080-2f0457a97753
+  type: SingleChoice
+  content: What is the main purpose of DeFi?
+  hint: Purpose of DeFi in the current scenario
+  explanation: Decentralised Finance is responsible for providing financial solutions for existing problems in the centralised system using blockchain.
+  answerKeys:
+    - B
+  subTopics:
+    - "introduction-to-decentralised-finance"
+  difficultyLevel: "Low"
+  choices:
+    - content: Technological playground for developers.
+      key: A
+      order: 0
+    - content: Providing financial solutions on blockchain.
+      key: B
+      order: 1
+    - content: No dedicated purpose
```

Comment:
The choices are not good. We could add choices from TradFi as they won't apply.

Reply:
Made changes in the options